src/subscription/cart/legal/SubscriptionLegal.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Hyperlink } from '@edx/paragon';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+// import LocalizedPrice from '../../../payment/cart/LocalizedPrice';
+import messages from './messages';
+
+const SubscriptionLegal = ({
```

Comment:
Is there a loading state and skeleton for this component?

Reply:
my guess is; we should not have loading states for static content. However if we really need to replicate the actual content with the page loading skeleton I can update the page skeleton in my next PR.

Reply:
Let's follow what UX has in the loading mock ups, which is one rectangle for this section.